src/sql/workbench/contrib/dashboard/browser/widgets/explorer/explorerTable.ts

```diff
@@ -204,11 +207,20 @@ export class ExplorerTable extends Disposable {
 					id: property.value,
 					field: property.value,
 					name: property.displayName,
-					width: columnWidth
+					width: columnWidth,
+					minWidth: 50
 				};
 			}
 		});
-		columns.push(this._actionsColumn.definition);
+		const actionsColumnDefinition = {
```

Comment:
You've already updated the width of the actionsColumn above, why do we need to create a clone of it here?

Reply:
Forgot to remove the width in above section, will take out. Reason needing to clone is because the definition does not use the defined width property unless the property showtext is true. Since the definition is also has resizable set to false, I decided to make a copy with desired column width that would show the column name.

Reply:
let me take a look at the code.

Reply:
ok, I think the right thing to do here is to modify the buttoncolumn.plugin.ts, to make it allow passed in width and also allow user define whether it is resizable, 50px probably works for 'Actions' in English, but might not be wide enough for other languages, so at least we can make it resizable.

```TypeScript
export class ButtonColumn<T extends Slick.SlickData> extends BaseClickableColumn<T> {

	constructor(private options: ButtonColumnOptions) {
		super();
	}

	public get definition(): Slick.Column<T> {
		return {
			id: this.options.id || this.options.title || this.options.field,
			width: this.options.width ?? 26,
			formatter: (row: number, cell: number, value: any, columnDef: Slick.Column<T>, dataContext: T): string => {
				const iconValue = getIconCellValue(this.options, dataContext);
				const escapedTitle = escape(iconValue.title ?? '');
				const iconCssClasses = iconValue.iconCssClass ? `codicon icon slick-plugin-icon ${iconValue.iconCssClass}` : '';
				const buttonTypeCssClass = this.options.showText ? 'slick-plugin-button slick-plugin-text-button' : 'slick-plugin-button slick-plugin-image-only-button';
				const buttonText = this.options.showText ? escapedTitle : '';
				return `<button tabindex=-1 class="${iconCssClasses} ${buttonTypeCssClass}" title="${escapedTitle}" aria-label="${escapedTitle}">${buttonText}</button>`;
			},
			name: this.options.name,
			resizable: this.options.resizable,
			selectable: false
		};
	}
}
````